addons/l10n_ch/static/src/scss/report_isr.scss

```diff
@@ -1,6 +1,6 @@
 @font-face {
     font-family: ocrb;
-    src: url('../font/ocrb.ttf') format('truetype');
+    src: url('../font/ocrb.otf') format('opentype');
```

Comment:
That doesn't seem very useful if we're just swapping the ttf or the otf? Unless the issue is that the ttf version is dodgy licensing-wise?

Also assigning the copyright to Matthew Skala seems a bit dodgy as it's apparently a conversion of the Metafont version, which was entirely recoded by Norbert Schwarz (according to http://ima.gob.pa/reportes/mpdf/ttfonts/ocrbinfo.txt)

Reply:
I'm using the one from [Matthew Skala](https://tsukurimashou.osdn.jp/ocr.php.en) since it's the one [provided by Debian ](https://packages.debian.org/buster/fonts-ocr-b) and that the previous one was not clearly licensed.

Also, here is an excerpt of the explanation PDF from Matthew Skala version:

> The version in this package descends from a set of Metafont
definitions by Norbert Schwarz of Ruhr-Universitaet Bochum,
bearing dates ranging from 1986 to 2010. He originally
distributed it under a "non-commercial use only"
restriction but has since released it for unrestricted use
and distribution. See the README file for more details.

Also, I'll make that PR description clearer in the way that I try to fix Lintian warnings like:
`W: odoo: truetype-font-prohibits-installable-embedding [never embed] usr/lib/python3/dist-packages/odoo/addons/l10n_ch/static/src/font/ocrb.ttf`

So, by using the Debian one, we will facilitate the replacement of the embedded font by a symlink to the Debian provided font and we will ensure the proper sync between the package behavior and the vanilla Odoo.

Reply:
> I'm using the one from Matthew Skala since it's the one provided by Debian and that the previous one was not clearly licensed.

Fair enough, that has no reason not to satisfy debian then.